### PR TITLE
Update Curtain.java

### PR DESCRIPTION
**Pull request by tprather12**

### CODE REVIEW

File: Curtain.java [hunk 1]
```diff
@@ -2,15 +2,23 @@
...
-        return true;
+        return curtain;
     }
     public void toggleCurtainClosed()
     {
+        if(curtain)
+        curtain = false;
+        
+        else 
+        curtain = true;
```
Comment (nwestervelt) [inline Curtain.java:22]: The indentation of these lines is a little off.
```Suggestion
            curtain = false;

        else 
            curtain = true;
```

**Review by nwestervelt — CHANGES_REQUESTED** [review]
Please see my comment about the indentation.

**Review by nwestervelt — APPROVED** [review]

**Review by GidJHock — APPROVED** [review]

**Comment by nwestervelt** [comment]
I merged the main branch onto this one, and now we need two more reviews. I should've done it earlier, or merged it anyways to see what would happen (I think it would've automatically fixed the problem).

**Review by nwestervelt — APPROVED** [review]

**Comment by nwestervelt** [comment]
I forgot I had it setup to not allow people who have committed on the branch to approve the PR. It's my fault that my own approval was invalidated, so I'm going to force merge this one.

**Pull request merged**